App.jsx

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+import List from './List'
+import Dashboard from './Dashboard'
+
+class App extends Component {
+  constructor() {
```

Comment:
you should pass props in here
```suggestion
  constructor(props) {
```